src/components/BulkEnrollmentPage/ConnectedPagination.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Potentially relevant: is there no test related to the functionality of the hitsPerPage/ page Size values?

Reply:
This component doesn't deal with hits per page or page size at all. All it does is call algolia if the table's page number is different from Algolia's page number.
Worth checking if my tests on the SearchResults component have tests for those things, tho.